deployment/src/main/java/io/quarkiverse/tekton/deployment/TektonConfiguration.java

```diff
@@ -10,6 +12,18 @@
 @ConfigRoot(phase = BUILD_TIME)
 public interface TektonConfiguration {
 
+    /**
+     * PipelineRun arguments to be passed as parameters
+     */
+    PipelineRun pipelinerun();
+
+    interface PipelineRun {
+        /**
```

Comment:
We might want to add a flag to enable / disable the generation of `PipelineRun` and `TaskRun`.